create.go

```diff
@@ -146,20 +143,21 @@ func homeDir() string {
 	return os.Getenv("USERPROFILE") // windows
 }
 
-func installCRDs(resources []spec.Schema, providerName, providerVersion string) {
-	var kubeconfig *string
-	if home := homeDir(); home != "" {
-		kubeconfig = flag.String("kubeconfig", filepath.Join(home, ".kube", "config"), "(optional) absolute path to the kubeconfig file")
-	} else {
-		kubeconfig = flag.String("kubeconfig", "", "absolute path to the kubeconfig file")
-	}
+func getK8sClientConfig() *rest.Config {
+	home := homeDir()
 
 	// use the current context in kubeconfig
-	clientConfig, err := clientcmd.BuildConfigFromFlags("", *kubeconfig)
+	clientConfig, err := clientcmd.BuildConfigFromFlags("", filepath.Join(home, ".kube", "config"))
```

Comment:
Not worth doing now, but we might want to also handle env var (e.g. `KUBECONFIG`) in future

Reply:
Agree, added now.